### ISSUE TITLE
Better, more verbose custom taxonomies

### ISSUE THREAD

**Issue by roikles** [roikles/Flexbones#63]
Found an issue where custom taxonomies were using some default syntax but it can be fixed with the following arguments for the taxonomy

```php
$args = array(
        'hierarchical'      => false,
        'labels'            => array(
            'name'              => $plural_name,
            'singular_name'     => $single_name,
            'menu_name'         => $plural_name,
            'all_items' => 'All ' . $plural_name,
            'edit_item' => 'Edit ' . $plural_name,
            'view_item' => 'View ' . $plural_name,
            'update_item' => 'Update ' . $single_name,
            'add_new_item' => 'Add new ' . $single_name,
            'new_item_name' => 'Add new ' . $single_name . ' name',
            'search_items' => 'Search ' . $plural_name,
            'popular_items' => 'Popular ' . $plural_name,
            'separate_items_with_commas' => 'Seperate ' . $plural_name . ' with commas',
            'add_or_remove_items' => 'Add or remove  ' . $plural_name,
            'choose_from_most_used' => 'Choose from most used ' . $plural_name,
            'not_found' => 'No ' . $plural_name . ' found.'
        ),
        'show_ui'           => true,
        'show_in_rest'      => true,
        'show_admin_column' => true,
        'rest_base'         => 'product_type',
    );
```

**Comment by DrizzlyOwl** [comment]
Shouldn't `rest_base` be a variable?

**Comment by roikles** [comment]
@DrizzlyOwl yup... it sure should.

```php
// other vars are alreadin the current file
$rest_base = 'product_type';
$args = array(
        'hierarchical'      => false,
        'labels'            => array(
            'name'              => $plural_name,
            'singular_name'     => $single_name,
            'menu_name'         => $plural_name,
            'all_items' => 'All ' . $plural_name,
            'edit_item' => 'Edit ' . $plural_name,
            'view_item' => 'View ' . $plural_name,
            'update_item' => 'Update ' . $single_name,
            'add_new_item' => 'Add new ' . $single_name,
            'new_item_name' => 'Add new ' . $single_name . ' name',
            'search_items' => 'Search ' . $plural_name,
            'popular_items' => 'Popular ' . $plural_name,
            'separate_items_with_commas' => 'Seperate ' . $plural_name . ' with commas',
            'add_or_remove_items' => 'Add or remove  ' . $plural_name,
            'choose_from_most_used' => 'Choose from most used ' . $plural_name,
            'not_found' => 'No ' . $plural_name . ' found.'
        ),
        'show_ui'           => true,
        'show_in_rest'      => true,
        'show_admin_column' => true,
        'rest_base'         => $rest_base,
    );
```

**Comment by roikles** [comment]
Fixed in #846dcdc
